src/tox/venv.py

```diff
@@ -658,7 +657,7 @@ def tox_testenv_create(venv, action):
         args.append("--system-site-packages")
     if venv.envconfig.alwayscopy:
         args.append("--always-copy")
-    if NO_DOWNLOAD:
+    if not venv.envconfig.download:
         args.append("--no-download")
```

Comment:
this requires a new-enough `virtualenv` -- the older option was `--never-download` (if you want further compatibility) -- I believe `--no-download` was added in virtualenv 14.x (lol, says me to _the current maintainer of virtualenv_ 😆 )

Reply:
indeed, I'm tempted to just require at least 14.0 for virtualenv 👍 three/four year-old one seems a sane requirement

Reply:
pre-commit requires >=15.2, I think it's probably fine for `tox` to require modern things too :D